Content.Server/_Funkystation/Atmos/AtmosResinDespawnSystem.cs

```diff
@@ -0,0 +1,75 @@
+using Content.Server._Funkystation.Atmos.Components;
+using Robust.Shared.Spawners;
+using Content.Shared.Atmos;
+using Content.Server.Atmos.EntitySystems;
+
+namespace Content.Server._Funkystation.Atmos.EntitySystems;
+
+
+/// <summary>
+/// ATMOS - Extinguisher Nozzle
+/// Sets atmospheric temperature to 20C and removes all toxins. 
+/// </summary>
+public sealed class AtmosResinDespawnSystem : EntitySystem
+{
+    [Dependency] private readonly AtmosphereSystem _atmo = default!;
+    [Dependency] private readonly GasTileOverlaySystem _gasOverlaySystem = default!;
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<AtmosResinDespawnComponent, TimedDespawnEvent>(OnDespawn);
+    }
+
+    private void OnDespawn(EntityUid uid, AtmosResinDespawnComponent comp, ref TimedDespawnEvent args)
+    {
+        if (!TryComp(uid, out TransformComponent? xform))
+            return;
+
+        var mix = _atmo.GetContainingMixture(uid, true);
+        GasMixture tempMix = new();
+        if (mix is null) return;
+
+        var gasesToRemove = new[]
+        {
+            Gas.CarbonDioxide,
+            Gas.Plasma,
+            Gas.Tritium,
+            Gas.Ammonia,
+            Gas.NitrousOxide,
+            Gas.Frezon,
+            ///Gas.BZ,
+            ///Gas.Healium,
+            ///Gas.Nitrium,
+            ///Gas.Hydrogen,
+            ///Gas.HyperNoblium,
+            ///Gas.ProtoNitrate,
+            ///Gas.Zauker,
+            ///Gas.Halon,
+            ///Gas.Helium,
+            ///Gas.AntiNoblium
+        };
```

Comment:
move this to the initializer and build it from every gas exluding oxygen and nitrogen so it automatically removes any gases added in the future